src/components/UserCard/UserCard.tsx

```diff
@@ -22,6 +28,7 @@ const UserCard = (props) => {
       >
         {t('visitProfile')}
       </a>{' '}
+      {/* @ts-expect-error resolve-fa-types */}
```

Comment:
Any context on this error? Looks like something that will repeat through components

Reply:
Update: should be solved after upgrading the library version